services/browsers/README.md

```diff
@@ -0,0 +1,48 @@
+Browsers Service
+================
+
+Browser service provides read/write API for browser object.
+
+To enable include browsers config part and add `browsers` to APIModules:
+
+
+```json
+{
+  "BrowsersConfig": {
+    "Enabled": true,
+  },
+  APIModules: "browsers"
+}
+```
+
+API
+---
+
+Enabling service will expose three additional methods:
+
+#### browsers_addBrowser
+
+Stores browser in the database.
+All fields are specified below:
+
+```json
+{
+  "id": "1",
+  "name": "first",
+  "timestamp": "10,
```

Comment:
missing `"`

Reply:
actually the first one should be removed, integer marshalled without braces